libs/language-server/src/lib/constraint/constraint-registry.ts

```diff
@@ -2,18 +2,28 @@
 //
 // SPDX-License-Identifier: AGPL-3.0-only
 
+import { ConstraintMetaInformation } from '../meta-information';
 import { registerMetaInformation } from '../meta-information/meta-inf-registry';
+import { ConstructorClass } from '../util';
 
 import { AllowlistConstraintMetaInformation } from './allowlist-constraint-meta-inf';
 import { DenylistConstraintMetaInformation } from './denylist-constraint-meta-inf';
 import { LengthConstraintMetaInformation } from './length-constraint-meta-inf';
 import { RangeConstraintMetaInformation } from './range-constraint-meta-inf';
 import { RegexConstraintMetaInformation } from './regex-constraint-meta-inf';
 
+export function getConstraintMetaInf(): ConstructorClass<ConstraintMetaInformation>[] {
```

Comment:
Consider to export a constant `DefaultConstraintMetaInfs` or so instead.

Reply:
I thought about it, but choose this design in order to be more consistent with `JayveeLangExtension#getBlockMetaInf`